IngerML/lab_2/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <cstdlib>
+#include <ctime> 
+using namespace std;
+const int razm = 10;
+int main()
+{
+	int i;
+	cout << "Mass 1:";
+	srand(time(0));
+	{
+	int arr[razm];
+	for (i = 0; i < razm; i++)
+	{
+		arr[i] = rand() % 100;
+		cout << arr[i] << ' ';
+	}
+	cout << endl;
+	int T = 0;
+	int a = 0;
+	for (int i = 1; i < razm; i++)
+	{
+		T = arr[i];
+		a = i - 1;
+		while ((a >= 0) && (arr[a] > T))
+		{
+			arr[a + 1] = arr[a];
+			a = a - 1;
+			arr[a + 1] = T;
+		}
+	}
+	cout << "Mass 2:";
+	for (i = 0; i < razm; i++) { cout << arr[i] << ' ';	}
+	cout << endl;
+
+	int min = 101;

```

Comment:
Well, in your code the largest possible element is 99, but what if it would be the largest INT possible? In that case you wouldn't be able to declare int variable that contains the higher bound. Find the way to count MAX and MIN elements without knowing their ranges